### DISCUSSION TITLE
not working as expected!

### DISCUSSION THREAD

**Discussion by EQNish** [cameroncros/OctoPrint-DiscordRemote#235] [Q&A]
I have the Bot set up, it is sending status of prints, but I can not make a request of the bot, nor send any commands too it!

So, my initial question would be why?
one thing of not, I do not have the "Base url" set, as my printers are not accessible from the Internet

**Comment by cameroncros** [comment]
Its probably not base URL, its most likely that your bot doesnt have enough permissions to read messages from your server. I havent set up a bot in a long time, so I can't help a lot.

Follow: https://www.upwork.com/resources/how-to-make-discord-bot

And try tweaking the permissions until it works?

**Comment by cameroncros** [comment]
https://github.com/cameroncros/OctoPrint-DiscordRemote/issues/226#issuecomment-1245165429
